### PR TITLE
Partial Home #24 closes #42, closes #43, closes #23, closes #21

### LINKED ISSUE #21: build Navbar
#24

### LINKED ISSUE #23: build Footer
#24 this must be done before hand

### LINKED ISSUE #42: create component ALL PRODUCTS
#24

### LINKED ISSUE #43: create endpoint to render ALL PRODUCTS
#24

### PR DESCRIPTION
**Pull request by cylau1031**
Basic  index app setup and AllProducts Component. Homepage incomplete

### CODE REVIEW

File: client/components/AllProducts.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+export const AllProducts = (props) => {
+  const {allProducts} = props
+  console.log(props)
```
Comment (knpt) [inline client/components/AllProducts.js]: remove console.log (sorry lol)

File: client/components/AllProducts.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  )
+}
+
+const mapStateToProps = state => {
+  return {
+    allProducts: state.products
+  }
+}
+
+const Container = connect(mapStateToProps)(AllProducts)
```
Comment (nicoleivanov) [inline client/components/AllProducts.js]: Please name the container something specific to this component, e.g. AllProductsContainer :)

File: client/reducers/allProducts.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+// ACTION TYPES
```
Comment (nicoleivanov) [inline client/reducers/allProducts.js]: Kim and I think that "getAllProducts" and "GET_ALL_PRODUCTS" would be more clear than setAllProducts, etc.

File: client/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import allProductsReducer from './allProducts'
+
+export default combineReducers({
+  products: allProductsReducer
+})
+
+export * from './allProducts'
```
Comment (nicoleivanov) [inline client/reducers/index.js]: Do we need to import * from './allProducts' in this file in order to be able to export *?

Comment (cylau1031) [reply client/reducers/index.js]: Don't need to explicitly import from './allProducts' in this file. Should work as is.

**Pull request merged**